### DISCUSSION TITLE
In latest Pimcore 6.9.1, the pdf preview is not working on grid

### DISCUSSION THREAD

**Discussion by EliShklovsky** [pimcore/pimcore#8789] [Q&A / Support]
Hi,
This is a continue from thread:
[https://talk.pimcore.org/t/in-latest-pimcore-6-4-the-pdf-preview-is-not-working-on-grid/3149](url)
It seems that this problem still occurs on 6.9.1
When uploading a pdf to the assets, it does not show a thumb preview on the grid or on the hover.
This can be seen on Pimcore official demo:
https://demo.pimcore.com/

One thing though that should be take care is ImageMagick security policy that in some cases blocking conversion,
this should be edited on policy.xml
This worked for me on .ai or .psd files but not for pdf

Any suggestions?
Thanks
Eli

**Comment by ACEHal9k** [comment]
I've the same problem, additionally preview of video files (MP4) is broken.
I also checked the policy.xml for ImageMagick, the permissions are set to "read | write" - seems not be the source of error.

- Rollback to 6.8.12 is working.
- Repeated upgrade to 6.9.1 --> again same preview problem
- Now I downgraded to 6.8.12 again.

The System-Requirements Check for version 6.8.12 shows me everything with a green check:
![Clipboard02](https://user-images.githubusercontent.com/40596957/115017820-5fe55180-9eb7-11eb-9162-9038cf8a94dd.png)

Version 6.9.1
![Clipboard03](https://user-images.githubusercontent.com/40596957/115018261-eef26980-9eb7-11eb-95e0-864d1219ad75.png)


System-Requirements Check on command line tells me everything is OK.

**Comment by brusch** [comment]
@EliShklovsky @ACEHal9k Previews are not generated as part of the maintenance and not on the fly anymore. 
See also #7363
